R/DeeDeeExperiment.R

```diff
@@ -208,12 +217,12 @@ DeeDeeExperiment <- function(se = NULL,
         )
       }
     } else if (is(this_de, "MArrayLM")) {
-      input_limma <- .importDE_limma(se_out, this_de, i)
-      se_out <- input_limma$se
+      input_limma <- .importDE_limma(sce_out, this_de, i)
+      sce_out <- input_limma$se
```

Comment:
here and in similar spots: you might want to call the element of the returned list also `sce` instead of `se`? I will try to pinpoint where in the .import functions

Reply:
a little bit here for example

<img width="595" alt="image" src="https://github.com/user-attachments/assets/9166ecb2-bf53-471b-907f-f9488fadf320" />